Enigmatry.Entry.TemplatingEngine.Razor/Enigmatry.Entry.TemplatingEngine.Razor.csproj

```diff
@@ -4,9 +4,10 @@
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <IsPackable>true</IsPackable>
-    <PackageId>Enigmatry.Entry.TemplatingEngine</PackageId>
+    <PackageId>Enigmatry.Entry.TemplatingEngine.Razor</PackageId>
```

Comment:
We need to deprecate the old project on nuget, so people get the warning that they need to update.